src/state/slices/portfolioSlice/utils.ts

```diff
@@ -45,7 +46,13 @@ import type {
 } from './portfolioSliceCommon'
 import { initialState } from './portfolioSliceCommon'
 
-export const chainIds = [ethChainId, btcChainId, cosmosChainId, osmosisChainId] as const
+export const chainIds = [
```

Comment:
this is non exhaustive and concerning

Reply:
yeah this whole thing should be deleted, this was pre chain-agnosticism and is just wrong